build/README.md

```diff
@@ -44,6 +44,12 @@ There are 3 different containers instances that are run from this image.  The fi
 
 All Docker names are suffixed with a hash derived from the file path (to allow concurrent usage on things like CI machines) and a version number.  When the version number changes all state is cleared and clean build is started.  This allows the build infrastructure to be changed and signal to CI systems that old artifacts need to be deleted.
 
+## Build artifacts
```

Comment:
we don't need to do this now, but noting that it probably makes sense to introduce the high level concept that we have `./_output/` (which ... is unusual to say the least), and then have sub sections on the different things you can find under here.

Reply:
+1 let's land this and @oz123 please file a follow up PR for @BenTheElder 's suggestion. thanks!